tests/neptune/new/test_handler.py

```diff
@@ -340,6 +340,35 @@ def test_assign_dict(self):
         self.assertEqual(exp['params/toys'].fetch_last(), "hat")
         self.assertEqual(exp['params/nested/nested/deep_secret'].fetch_last(), 15)
 
+    def test_fetch_dict(self):
```

Comment:
Please make sure we have good tests for assigning dict as well.

Reply:
well, we have "some" tests ;) I'm more that happy to add more, but outside of scope of this MR?